kratos/utilities/compare_elements_and_conditions_utility.h

```diff
@@ -39,39 +38,19 @@ class CompareElementsAndConditionsUtility
     ///@name Operations
     ///@{
 
-    static void GetRegisteredName(const Element& rElement, std::string& rName) {
-        KRATOS_TRY;
+    static void KRATOS_API(KRATOS_CORE) GetRegisteredName(const Geometry<Node<3>>& rGeometry, std::string& rName);
```

Comment:
```suggestion
    static void KRATOS_API(KRATOS_CORE) GetRegisteredName(
        const Geometry<Node<3>>& rGeometry,
        std::string& rName);
```

(minor style guide suggestion)